src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -28,20 +28,25 @@ import withOrganization from 'app/utils/withOrganization';
 import Access from 'app/components/acl/access';
 import ConfigStore from 'app/stores/configStore';
 import GlobalSelectionHeader from 'app/components/organizations/globalSelectionHeader';
+import {promptsUpdate} from 'app/actionCreators/prompts';
 
 import {Incident} from '../types';
 import {TableLayout, TitleAndSparkLine} from './styles';
 import AlertListRow from './row';
 
 const DEFAULT_QUERY_STATUS = 'open';
 
+const DOCS_URL =
+  'https://docs.sentry.io/workflow/alerts-notifications/alerts/?_ga=2.21848383.580096147.1592364314-1444595810.1582160976';
```

Comment:
What is this querystring for?

Reply:
Not sure, copied from where it was before. @adhiraj ?

Reply:
Not sure, we can remove it